dev/ledger.md

```diff
@@ -494,6 +499,14 @@ how many units of that asset is held by the account and a boolean flag
 indicating if the holding is frozen or unfrozen.  An account that holds
 any asset cannot be closed.
 
+# Participation Updates
+
+Participation updates contains a single list of addresses of accounts which 
+have been deemed to be _expired_. An account is said to be expired when the last
+valid vote round in its participation key is strictly less than the current round
+that is being processed.  Once included in this list, an account will be marked 
+offline as part of the validation of this block.
```

Comment:
...
Once included in this list, an account will be marked 
offline as part of applying the block changes to the ledger.